packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/RequirementSubForm/RequirementSubForm.tsx

```diff
@@ -24,15 +25,15 @@ const RequirementSubForm = ({
 }: RequirementSubFormType) => {
   const [requirementType, setRequirementType] = useState('');
   const isTokenRequirement = Object.values(TOKENS).includes(requirementType);
-  const is1155Requirement = requirementType === SPECIFICATIONS.ERC_1155;
+  const is1155Requirement = requirementType === ERC_SPECIFICATIONS.ERC_1155;
   const isCosmosRequirement =
     requirementType === TOKENS.COSMOS_TOKEN ||
-    requirementType === SPECIFICATIONS.CW_721;
+    requirementType === CW_SPECIFICATIONS.CW_721;
   const helperTextForAmount = {
     [TOKENS.EVM_TOKEN]: 'Using 18 decimal precision',
     [TOKENS.COSMOS_TOKEN]: 'Using 6 decimal precision',
-    [SPECIFICATIONS.ERC_20]: 'Using 18 decimal precision',
-    [SPECIFICATIONS.ERC_721]: '',
+    [ERC_SPECIFICATIONS.ERC_20]: 'Using 18 decimal precision',
+    [ERC_SPECIFICATIONS.ERC_721]: '',
   };
```

Comment:
not sure but does CW_721 use some specific decimal precision? If it does then we might have to add helper text for that in  `helperTextForAmount`

Reply:
No, it's same as ERC_721. I added an empty string to follow the pattern, in case we want to update it later.

Reply:
gotcha thanks, resolving.